native/src/main/java/io/ballerina/stdlib/grpc/logging/GrpcLogManager.java

```diff
@@ -0,0 +1,168 @@
+            } catch (IOException e) {
+                throw new RuntimeException("failed to setup GRPC trace log file: " + logFilePath.getValue(), e);
+            }
+        }
+
+        BString host = traceLogAdvancedConfig.getStringValue(GRPC_TRACE_LOG_HOST);
+        Long port = traceLogAdvancedConfig.getIntValue(GRPC_TRACE_LOG_PORT);
+        if ((host != null && !host.getValue().trim().isEmpty()) && (port != null && port != 0)) {
+            try {
+                SocketHandler socketHandler = new SocketHandler(host.getValue(), port.intValue());
+                socketHandler.setFormatter(new JsonLogFormatter());
+                socketHandler.setLevel(Level.FINEST);
+                grpcTraceLogger.addHandler(socketHandler);
+                traceLogsEnabled = true;
+            } catch (IOException e) {
+                throw new RuntimeException("failed to connect to " + host.getValue() + ":" + port.intValue(), e);
+            }
+        }
+
+        if (traceLogsEnabled) {
+            grpcTraceLogger.setLevel(Level.FINEST);
+            System.setProperty(HTTP_TRACE_LOG_ENABLED, "true");
+            stdErr.println("ballerina: GRPC trace log enabled");
+        }
+    }
+
+    /**
+     * Initializes the GRPC access logger.
+     */
+    public void setHttpAccessLogHandler(BMap accessLogConfig) {
+
+        if (grpcAccessLogger == null) {
+            // keep a reference to prevent this logger from being garbage collected
+            grpcAccessLogger = Logger.getLogger(HTTP_ACCESS_LOG);
+        }
+        PrintStream stdErr = System.err;
+        boolean accessLogsEnabled = false;
+
+        Boolean consoleLogEnabled = accessLogConfig.getBooleanValue(GRPC_LOG_CONSOLE);
+        if (consoleLogEnabled) {
+            ConsoleHandler consoleHandler = new ConsoleHandler();
+            consoleHandler.setFormatter(new GrpcAccessLogFormatter());
+            consoleHandler.setLevel(Level.INFO);
+            grpcAccessLogger.addHandler(consoleHandler);
+            grpcAccessLogger.setLevel(Level.INFO);
+            accessLogsEnabled = true;
+        }
+
+        BString filePath = accessLogConfig.getStringValue(GRPC_LOG_FILE_PATH);
+        if (filePath != null && !filePath.getValue().trim().isEmpty()) {
+            try {
+                FileHandler fileHandler = new FileHandler(filePath.getValue(), true);
+                fileHandler.setFormatter(new GrpcAccessLogFormatter());
+                fileHandler.setLevel(Level.INFO);
+                grpcAccessLogger.addHandler(fileHandler);
+                grpcAccessLogger.setLevel(Level.INFO);
+                accessLogsEnabled = true;
+            } catch (IOException e) {
+                throw new RuntimeException("failed to setup GRPC access log file: " + filePath.getValue(), e);
+            }
+        }
+
+        if (accessLogsEnabled) {
+            System.setProperty(HTTP_ACCESS_LOG_ENABLED, "true");
+            stdErr.println("ballerina: GRPC access log enabled");
```

Comment:
```suggestion
            stdErr.println("ballerina: gRPC access log enabled");
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/413/commits/303299de9612deb38959f15b7389ed952825454a